ansible_collections/arista/avd/roles/cv_deploy/tasks/main.yml

```diff
@@ -32,6 +32,9 @@
       description: "{{ cv_workspace_description | arista.avd.default }}"
       requested_state: "{{ 'submitted' if cv_submit_workspace else 'built' }}"
       force: "{{ cv_submit_workspace_force }}"
+      build_warnings: "{{ cv_workspace_build_warnings.enabled | arista.avd.default(true) }}"
+      build_warnings_suppress_patterns: "{{ cv_workspace_build_warnings.suppress_patterns | arista.avd.default([]) }}"
+      build_warnings_suppress_portfast: "{{ cv_workspace_build_warnings.suppress_portfast | arista.avd.default(false) }}"
```

Comment:
you don't need the default filter here since you already set them in the role defaults.

Reply:
fixed